### ISSUE TITLE
Add a "dev-only" boolean to items

### ISSUE THREAD

**Issue by msouth** [ironbane/IronbaneServerLegacy#422]
(This is partly in reference to some of the conversation in #416 .)

Feature:  a boolean characteristic on an item that marks it dev-only.  If the boolean is on, the item can't be picked up by a non-dev player, even if it's dropped where they can reach it.

If you make items that aren't supposed to be in the game yet dev only, then the devs don't have to decide whether to give in and give them to someone.  The decision is, "Do I make this person a dev, or not?"  

If they're not a dev, you can drop the item, but the non-dev can't pick it up.  People could still be taken on field trips to preview/test new areas, but they won't be able to pick up any drops that aren't already blessed to be held by non-devs.

**Comment by ghost** [comment]
Agreed, this would be nice to have. Prevents even bank glitches from providing banked dev items.
